src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -2268,7 +2262,8 @@ private MemberDeclarationSyntax ParseMemberDeclarationOrStatementCore(SyntaxKind
                     _termState = saveTerm;
                     if (statement != null)
                     {
-                        return _syntaxFactory.GlobalStatement(statement);
+                        return _syntaxFactory.GlobalStatement(
```

Comment:
>GlobalStatement [](start = 46, length = 15)

Would it be worthwhile to add a `SyntaxFactory.GlobalStatement` method that doesn't take the two extra arguments?

Reply:
Indeed!